nxdrive/engine/dao/migrations/migration.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from sqlite3 import Cursor
+
+
+class MigrationInterface(ABC):
+    """Migration interface inherited by all migrations."""
+
+    @abstractmethod
+    def upgrade(self, cursor: Cursor) -> None:
+        raise NotImplementedError
```

Comment:
For `abstractmethod`s there is no need for that, just let the method empty. Also, it would be a good thing to add docstrings to all those methods.